### PR TITLE
feat: add classification in profile FS-97

### PR DESCRIPTION
**Pull request by sb88k**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/FS-97" title="FS-97" target="_blank">Sub-taskFS-97</a>  [iOS] Show classified banner in user profile
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

Show security classification view in user profile and connection screens

### CODE REVIEW

**Review by johnxnguyen — CHANGES_REQUESTED** [review]
You'll need to update some snapshot tests.

File: Wire-iOS/Sources/UserInterface/ConnectRequests/IncomingConnectionView.swift [hunk 1]
```diff
@@ -122,7 +125,13 @@ final class IncomingConnectionView: UIView {
             userDetailView.centerXAnchor.constraint(equalTo: centerXAnchor),
             userDetailView.topAnchor.constraint(equalTo: usernameLabel.bottomAnchor, constant: 4),
             userDetailView.leftAnchor.constraint(greaterThanOrEqualTo: leftAnchor),
-            userDetailView.bottomAnchor.constraint(lessThanOrEqualTo: userImageView.topAnchor),
+            userDetailView.bottomAnchor.constraint(lessThanOrEqualTo: securityLevelView.topAnchor),
+
+            securityLevelView.centerXAnchor.constraint(equalTo: centerXAnchor),
+            securityLevelView.topAnchor.constraint(equalTo: userDetailView.bottomAnchor),
```
Comment (David-Henner) [inline Wire-iOS/Sources/UserInterface/ConnectRequests/IncomingConnectionView.swift:131]:
```suggestion
            userDetailView.bottomAnchor.constraint(lessThanOrEqualTo: securityLevelView.topAnchor),

            securityLevelView.centerXAnchor.constraint(equalTo: centerXAnchor),
```

Comment (David-Henner) [reply Wire-iOS/Sources/UserInterface/ConnectRequests/IncomingConnectionView.swift:131]: duplicate constraint

File: Wire-iOS/Sources/UserInterface/UserProfile/ProfileViewController.swift [hunk 2]
```diff
@@ -238,6 +242,7 @@ final class ProfileViewController: UIViewController {
         guard let tabsView = tabsController?.view else { fatal("Tabs view is not created") }
 
         usernameDetailsView.translatesAutoresizingMaskIntoConstraints = false
+        securityLevelView.translatesAutoresizingMaskIntoConstraints = false
         tabsView.translatesAutoresizingMaskIntoConstraints = false
         profileFooterView.translatesAutoresizingMaskIntoConstraints = false
         incomingRequestFooter.translatesAutoresizingMaskIntoConstraints = false
```
Comment (David-Henner) [inline Wire-iOS/Sources/UserInterface/UserProfile/ProfileViewController.swift:248]:
```suggestion
        [usernameDetailsView,
        securityLevelView,
        tabsView,
        profileFooterView,
        incomingRequestFooter].prepareForLayout()
```

**Review by johnxnguyen — COMMENTED** [review]
The design spec shows the security level view with a border only on the top and bottom edge, but the snapshots show a full border around all edges.

**Review by johnxnguyen — APPROVED** [review]
Looks good, you just need to update the dependencies since I merged my branches.

**Review by David-Henner — APPROVED** [review]

**Pull request merged**